app/controllers/projects_controller.rb

```diff
@@ -1,4 +1,5 @@
 class ProjectsController < ApplicationController
+  # before_filter :set_env_session
```

Comment:
Commented out code - do we want this in the commit as well?  It's the only thing updated in this file so it seems odd.